framework/widgets/BaseListView.php

```diff
@@ -77,7 +77,8 @@
      */
     public $showOnEmpty = false;
     /**
-     * @var string the HTML content to be displayed when [[dataProvider]] does not have any data.
+     * @var string|bool the HTML content to be displayed when [[dataProvider]] does not have any data. When `false`,
+     * row with empty text will not be displayed.
```

Comment:
@SilverFire Agree, it's a bit ambiguous. But this is a `BaseListView` which renders not a table. I guess this problem also needs to be checked for `BaseListView` too.